.circleci/config.yml

```diff
@@ -148,7 +148,7 @@ jobs:
           at: .
       - run:
           name: Generate documentation
-          command: pipenv run ./dev-tools/generate_documentation.sh
+          command: pipenv run ./dev-tools/generate_documentation.sh --clean
```

Comment:
The documentation build on CircleCI is clean every time, you don't need to pass this parameter.
I think the problem is that `linkcheck` does not have a Sphinx documentation and thus the automatic link targets do not work. You can fix this by adding the not found modules to `nitpick_ignore` in [sphinx/conf.py](https://github.com/Integreat/integreat-cms/blob/3d33bbee2867642b52fe712b4026b2f3554dee5f/sphinx/conf.py#L111-L135).